client/webserver/site/src/js/settings.js

```diff
@@ -46,6 +52,23 @@ export default class SettingsPage extends BasePage {
     Doc.bind(page.addCert, 'click', () => this.page.certFile.click())
     forms.bind(page.dexAddrForm, page.submitDEXAddr, () => { this.verifyDEX() })
     forms.bind(page.confirmRegForm, page.submitConfirm, () => { this.registerDEX() })
+    forms.bind(page.authorizeAccountExportForm, page.authorizeExportAccountConfirm, () => { this.exportAccount() })
+
+    const exchangesDiv = page.exchanges
+    var exportAccountKeyButton
+    // eslint-disable-next-line no-unused-vars
+    for (const [host, xc] of Object.entries(app.user.exchanges)) {
```

Comment:
`for (const host of Object.keys(app.user.exchanges)) {`

Don't disable the linter, please.